test/test-get-passwd.c

```diff
@@ -0,0 +1,60 @@
+#include "uv.h"
```

Comment:
I've never added it to anything in this repo before, so I'm going with nay. If you want it here, I'd be glad to add it though.